change/eslint-plugin-minecraft-linting-ce20bfba-df62-44a8-b3e6-aaad8618fdb6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update setBlock to setblock",
```

Comment:
super nit: This comment goes into the changelog and the commit, so much appreciated if it also explains the rationale for this, something like `Change setBlock command suggestion to use command casing convention`.

Reply:
@rlandav Is it best to run change again and put a new comment, or can I just change the comment in the generated change file?